api/src/main/java/org/pmiops/workbench/actionaudit/targetproperties/BypassTimeTargetProperty.kt

```diff
@@ -7,5 +7,7 @@ constructor(override val propertyName: String) : SimpleTargetProperty {
     CT_COMPLIANCE_TRAINING("ct_compliance_training_time"),
     ERA_COMMONS("era_commons_bypass_time"),
     TWO_FACTOR_AUTH("two_factor_auth_bypass_time"),
+    PROFILE_CONFIRMATION("profile_confirmation_bypass_time"),
+    PUBLICATION_CONFIRMATION("publication_confirmation_bypass_time"),
```

Comment:
To the best of my knowledge, the specific strings passed here aren't used in our code again after being sent to the BigQuery audit table. Let me know if that's not the case.